### PR TITLE
update from template to secure-client-hub

### PR DESCRIPTION
**Pull request by shewood**
### Description
Update the teamcity builds for Secure Client Hub from next template

List of proposed changes:
Hopefully changes all of the location from Next Template to Secure Client Hub for the builds to work in the new team city instance. 

-
-

### CODE REVIEW

File: .github/workflows/pr-helper.yml [hunk 1]
```diff
@@ -25,6 +25,6 @@ jobs:
           edit-mode: replace
           issue-number: ${{ github.event.pull_request.number }}
           body: |
-            [Integration Deployment :rocket: ](https://${{ github.event.pull_request.base.repo.name }}-dyna-${{ github.head_ref }}.bdm-dev.dts-stn.com) - [Check build status](https://teamcity.dts-stn.com/buildConfiguration/Dev_NextTemplate_Build_Dynamic?branch=${{ github.head_ref }}&buildTypeTab=overview&mode=builds)
+            [Integration Deployment :rocket: ](https://${{ github.event.pull_request.base.repo.name }}-dyna-${{ github.head_ref }}.bdm-dev.dts-stn.com) - [Check build status](https://teamcity.dts-stn.com/buildConfiguration/Dev_SCH_Build_Dynamic?branch=${{ github.head_ref }}&buildTypeTab=overview&mode=builds)
```
Comment (s-laugh) [inline .github/workflows/pr-helper.yml:28]:
```suggestion
            [Integration Deployment :rocket: ](https://${{ github.event.pull_request.base.repo.name }}-dyna-${{ github.head_ref }}.dev.dts-stn.com) - [Check build status](https://teamcity.dev.admin.dts-stn.com/buildConfiguration/Dev_SCH_Build_Dynamic?branch=${{ github.head_ref }}&buildTypeTab=overview&mode=builds)
```

File: azure-pipelines.yml [hunk 2]
```diff
@@ -23,7 +23,7 @@ stages:
     - task: Docker@2
       inputs:
         containerRegistry: 'MTSContainers'
-        repository: 'next-template'
+        repository: 'secure-client-hub'
```
[the remark this answers is not in the document]
Comment (shewood) [reply azure-pipelines.yml:26]: OK

[the remark this answers is not in the document]
Comment (shewood) [reply azure-pipelines.yml:26]: SRE says we are still using azure for container registry.

[the remark this answers is not in the document]
Comment (s-laugh) [reply azure-pipelines.yml:26]: Yes, but we aren't using azure pipelines. Our container registry is added in the TC job.

**Review by s-laugh — APPROVED** [review]

**Pull request merged**